sklearn/manifold/_debugvars.h

```diff
@@ -0,0 +1,3 @@
+#define DEBUGFLAG 0
```

Comment:
Yeah, including an extra file just for compile-time debug flags is gnarly. Didn't think about enums before now, but using an enum is a much more elegant solution, and performance-wise seems to make no (significant) impact. Thanks -- I've made this change.